client/my-sites/email/email-management/home/email-plan.jsx

```diff
@@ -294,7 +294,7 @@ class EmailPlan extends React.Component {
 				<div className="email-plan__actions">
 					<VerticalNav>
 						<VerticalNavItem { ...addMailboxProps }>
-							{ translate( 'Add new mailbox' ) }
+							{ translate( 'Add new mailboxes' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Add New Mailboxes' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/fix/wrong-mailbox-types)

Reply:
I've made this change so this label matches the name of the page it leads to.

Reply:
Who can do more can do less 🙂. But anyway, I'll stand by my point as I think being consistent with the name of the page prevails here.

Reply:
Fair enough -- we can definitely change back if needed.